### ISSUE TITLE
Something wrong with lab key selling prices

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#152]
## **General information**
**Project issue is for:**
Dont really know

**Expected result**:
To sell green lab key for ~1.000.000

**Actual result**:
I sold it for ~10.000.000

So, i can buy green lab key for 8-9kk on flea and sell it for 10kk to therapist. Prices are broken for ALL lab keys (green, red...). Tested with and without mods (deleted "plugins" and "mods" with creating new easy start profile)

## **How to reproduce**
1. Find green, red... lab key on flea
2. Buy it
3. Sell to therapist

![image](https://dev.sp-tarkov.com/attachments/741e53a5-9835-4c26-aac4-0e04ffa9dd8f)

---
### Attachments
- [server-2024-09-08.log](https://dev.sp-tarkov.com/attachments/c1391101-b52a-4c75-a565-35fba2030ad9) (1462 KiB)
- [LogOutput.log](https://dev.sp-tarkov.com/attachments/c09afb5d-95b4-4789-adcb-4f4750771d14) (35 KiB)
- [2024.09.08_14-59-48_0.14.9.1.30626 traces.log](https://dev.sp-tarkov.com/attachments/257908ab-1214-4875-bb66-48bf66b98568) (44 KiB)
- [image.png](https://dev.sp-tarkov.com/attachments/741e53a5-9835-4c26-aac4-0e04ffa9dd8f) (1601 KiB)

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803) by _**DaytonFox**_

**Comment by refringe** [comment]
This is caused by changes BSG made to live that tanked the price of keycards, which is being imported via the Live Flea Prices mod. 

SPT uses the higher of either the listed flea price, or the Handbook price as the base price for items. In this case, while Live Flea Prices sets the flea price to 1.8 million, the handbook price for SPT 3.9.8 (And in live 0.14.9) is 11.8 million, resulting in the behaviour you're seeing

tl;dr: Not an SPT bug, BSG changed the price of keycards in live

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28127) by _**DrakiaXYZ**_

**Comment by refringe** [comment]
> This is caused by changes BSG made to live that tanked the price of keycards, which is being imported via the Live Flea Prices mod. 
> 
> SPT uses the higher of either the listed flea price, or the Handbook price as the base price for items. In this case, while Live Flea Prices sets the flea price to 1.8 million, the handbook price for SPT 3.9.8 (And in live 0.14.9) is 11.8 million, resulting in the behaviour you're seeing
> 
> tl;dr: Not an SPT bug, BSG changed the price of keycards in live

But flea prices still less than 10kk selling price, any fixes for this? And 10kk "problem" will be "fixed" will next updates?

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28129) by _**DaytonFox**_

**Comment by refringe** [comment]
The flea price for keycards is only less than the vendor sell price if you're running `Live Flea Prices` or some other mod that adjusts flea prices to be lower than vendor prices. If that happens, SPT utilizes the vendor price as a _base_ price, with a random up/down adjustment, which means items can end up cheaper than the vendor price

SPT with no flea price adjusting mods will not have this issue

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28130) by _**DrakiaXYZ**_

**Comment by refringe** [comment]
> The flea price for keycards is only less than the vendor sell price if you're running `Live Flea Prices` or some other mod that adjusts flea prices to be lower than vendor prices. If that happens, SPT utilizes the vendor price as a _base_ price, with a random up/down adjustment, which means items can end up cheaper than the vendor price
> 
> SPT with no flea price adjusting mods will not have this issue

do you know is this issue works only on lab keys?

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28131) by _**DaytonFox**_

**Comment by refringe** [comment]
It will happen on anything that has a lower current live flea price than its vendor price was in 0.14.9

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28132) by _**DrakiaXYZ**_

**Comment by refringe** [comment]
> It will happen on anything that has a lower current live flea price than its vendor price was in 0.14.9

thx. meanwhile i found another issue. I can sell items without "finded in raid" status. should i create new ticket (all logs will be same)?

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28133) by _**DaytonFox**_

**Comment by refringe** [comment]
> > It will happen on anything that has a lower current live flea price than its vendor price was in 0.14.9
> 
> thx. meanwhile i found another issue. I can sell items without "finded in raid" status. should i create new ticket (all logs will be same)?

This is how it is on live now too.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/803#issuecomment-28134) by _**Lacyway**_
